admin_guide/diagnostics_tool.adoc

```diff
@@ -115,3 +115,54 @@ A client with *cluster-admin* access available (for any user, but only the
 current master) should be able to diagnose the status of infrastructure such as
 nodes, registry, and router. In each case, running `oc adm diagnostics` looks
 for the client configuration in its standard location and uses it if available.
+
+[[additional-cluster-health-checks]]
+== Additional Diagnostic Checks via Ansible
+
+Some additional diagnostic checks are available through the https://hub.docker.com/r/openshift/openshift-ansible[openshift-ansible]
+Docker image.
+
+These checks belong to a diagnostic task meant to be run against the Ansible inventory file for a deployed OpenShift cluster.
+They can report common OpenShift-specific problems for the current deployment.
+
+To disable specific checks, include the variable `openshift_disable_check` with a
+comma-delimited list of check names in your inventory file. For example:
+
+----
+openshift_disable_check=ovs_version,etcd_volume
+----
+
+The following table describes available health checks that can be run on a deployed OpenShift cluster.
+
+[[diagnostic-checks]]
+.Diagnostic Checks
+[options="header"]
+|===
+
+|Check Name |Purpose
+
+|`*ovs_version*`
+|This check ensures that a host has the correct version of Open vSwitch installed for the currently deployed version of OpenShift.
+
+|`*kibana*`, `*curator*`, `*elasticsearch*`, `*fluentd*`
+|This set of checks verifies that `Elasticsearch`, `Fluentd`, and `Curator` pods have been deployed and are in a `running` state, and that a connection can be established between the control host and the exposed `Kibana` URL.
+These checks will only run if the `openshift_hosted_logging_deploy` inventory variable is set to `true`, to ensure that they are executed in a deployment where a logging stack has been deployed.
+
+|`*etcd_imagedata_size*`
+|This check measures the total size of OpenShift image data in an Etcd cluster. Fails if the calculated size exceeds a user-defined limit. If no limit is specified, this check will fail if the size of OpenShift image data amounts to `50%`
+or more of the currently used space in the Etcd cluster.
```

Comment:
Thanks, will make it lowercase. I don't think it should be wrapped in backticks to avoid confusion with logging component names